content/api/commands/attachfile.md

```diff
@@ -0,0 +1,235 @@
+---
+title: attachFile
+---
+
+Attaches a file or files to an HTML5 input element or simulates dragging a file or files into the browser.
+
+## Syntax
+
+```javascript
+.attachFile(file)
+.attachFile(file, options)
+.attachFile([file], options)
+.attachFile([file], options)
```

Comment:
I feel like the `.attachFile([file], options)` lines are unnecessary here

Reply:
There definitely shouldn't be two of them, I'll fix that.

Even though it's not strictly speaking a different signature (still 1-2 args), it seemed worth highlighting, right at the top, that you can pass in an array of files. Perhaps `.attachFile([file, file2, ...], options)` would be clearer in that intention? Or do you not think it's worth having additional lines here at all?

Reply:
I think we could do

```js
.attachFile([file, file2, ...], options)
```

or 

```js
.attachFile(fileArray, options)
```

but in either case, we probably also want to show the non-`options` variant, so maybe something like this?

```js
.attachFile(file)
.attachFile(file, options)
.attachFile(fileArray)
.attachFile(fileArray, options)
```

Reply:
Yeah, that's why there was a duplicate line, accidentally had `options` in both. Pushed up changes.